typescript/sdk/src/utils/gnosisSafe.js

```diff
@@ -83,9 +81,9 @@ export async function getSafe(chain, multiProvider, safeAddress) {
       [domainId]: {
         // Use the safe address for multiSendAddress and multiSendCallOnlyAddress
         // if the contract is not deployed or if the version is not found
-        multiSendAddress: multiSend?.defaultAddress || safeAddress,
+        multiSendAddress: multiSend?.networkAddresses[domainId] || safeAddress,
```

Comment:
iiuc, we expect our domainId to always "match" theirs?

Reply:
yup, because it's for EVM chains and we have chainId==domainId for those (and chainId is pretty standardised on evm)

Reply:
I'll add a comment to that effect in code